art/text_dic3.py

```diff
@@ -26432,3 +26432,973 @@
+    'w': '𝐰',
+    ':': ':',
+    'v': '𝐯',
+    '5': '𝟓',
+    'L': '𝐋'}
+
+fancy136_dic = {
+    'p': '𝗽',
+    'B': '𝗕',
+    'm': '𝗺',
+    'd': '𝗱',
+    'a': '𝗮',
+    '!': '❗',
+    's': '𝘀',
+    'u': '𝘂',
+    '}': '}',
+    '/': '/',
+    '>': '>',
+    ';': ';',
+    'J': '𝗝',
+    '(': '(',
+    'r': '𝗿',
+    'f': '𝗳',
+    '{': '{',
+    'K': '𝗞',
+    '-': '-',
+    'P': '𝗣',
+    ')': ')',
+    '%': '%',
+    '_': '_',
+    '9': '𝟵',
+    'l': '𝗹',
+    'H': '𝗛',
+    'C': '𝗖',
+    '$': '$',
+    ' ': ' ',
+    '\\': '\\',
+    'j': '𝗷',
+    '5': '𝟱',
+    'Q': '𝗤',
+    'k': '𝗸',
+    'b': '𝗯',
+    'I': '𝗜',
+    'x': '𝘅',
+    'y': '𝘆',
+    '.': '.',
+    '^': '^',
+    'L': '𝗟',
+    'v': '𝘃',
+    'T': '𝗧',
+    ']': ']',
+    '&': '&',
+    'R': '𝗥',
+    '[': '[',
+    'N': '𝗡',
+    '4': '𝟰',
+    '0': '𝟬',
+    '7': '𝟳',
+    'c': '𝗰',
+    '?': '❓',
+    'M': '𝗠',
+    'S': '𝗦',
+    'W': '𝗪',
+    'g': '𝗴',
+    '~': '~',
+    '<': '<',
+    'O': '𝗢',
+    'D': '𝗗',
+    '#': '#',
+    '"': '❞',
+    "'": '❜',
+    'i': '𝗶',
+    'U': '𝗨',
+    '|': '|',
+    '`': '`',
+    'E': '𝗘',
+    'F': '𝗙',
+    '8': '𝟴',
+    'A': '𝗔',
+    '@': '@',
+    '3': '𝟯',
+    'G': '𝗚',
+    'n': '𝗻',
+    'h': '𝗵',
+    '6': '𝟲',
+    'w': '𝘄',
+    '2': '𝟮',
+    ':': ':',
+    ',': ',',
+    '+': '+',
+    'q': '𝗾',
+    'V': '𝗩',
+    'e': '𝗲',
+    '*': '*',
+    'z': '𝘇',
+    'o': '𝗼',
+    'Y': '𝗬',
+    '=': '=',
+    'X': '𝗫',
+    '1': '𝟭',
+    'Z': '𝗭',
+    't': '𝘁'}
+
+fancy137_dic = {
+    '<': '<',
+    '@': '@',
+    'e': '𝖊',
+    '^': '^',
+    'M': '𝕸',
+    ')': ')',
+    '$': '$',
+    'H': '𝕳',
+    'W': '𝖂',
+    'w': '𝖜',
+    'c': '𝖈',
+    'Y': '𝖄',
+    't': '𝖙',
+    'h': '𝖍',
+    'm': '𝖒',
+    'v': '𝖛',
+    'T': '𝕿',
+    'E': '𝕰',
+    'D': '𝕯',
+    '|': '|',
+    '2': '2',
+    'K': '𝕶',
+    'A': '𝕬',
+    '}': '}',
+    '%': '%',
+    '/': '/',
+    'U': '𝖀',
+    'C': '𝕮',
+    '?': '❓',
+    '[': '[',
+    '6': '6',
+    '9': '9',
+    'l': '𝖑',
+    '*': '*',
+    ' ': ' ',
+    '"': '❞',
+    '`': '`',
+    'u': '𝖚',
+    ';': ';',
+    'P': '𝕻',
+    'X': '𝖃',
+    ':': ':',
+    '#': '#',
+    '!': '❗',
+    '0': '0',
+    '>': '>',
+    'y': '𝖞',
+    '_': '_',
+    ',': ',',
+    '5': '5',
+    's': '𝖘',
+    'o': '𝖔',
+    'x': '𝖝',
+    '+': '+',
+    'p': '𝖕',
+    '{': '{',
+    ']': ']',
+    'B': '𝕭',
+    'k': '𝖐',
+    'j': '𝖏',
+    'R': '𝕽',
+    'a': '𝖆',
+    'f': '𝖋',
+    'g': '𝖌',
+    'd': '𝖉',
+    'n': '𝖓',
+    '4': '4',
+    '.': '.',
+    'N': '𝕹',
+    '8': '8',
+    '-': '-',
+    '7': '7',
+    '\\': '\\',
+    'i': '𝖎',
+    '~': '~',
+    'J': '𝕵',
+    'I': '𝕴',
+    "'": '❜',
+    '(': '(',
+    'S': '𝕾',
+    '1': '1',
+    'b': '𝖇',
+    'F': '𝕱',
+    'Q': '𝕼',
+    'q': '𝖖',
+    '3': '3',
+    'Z': '𝖅',
+    'L': '𝕷',
+    'r': '𝖗',
+    'O': '𝕺',
+    '&': '&',
+    '=': '=',
+    'G': '𝕲',
+    'V': '𝖁',
+    'z': '𝖟'}
+
+tarty1_dic = {
```

Comment:
Isn't it better if we sort these lines respected to the alphabetic order (like non-ASCII fonts)?

Reply:
Which lines? Dictionary's keys?

Reply:
yes